packs/FotS.json

```diff
@@ -140,6 +189,35 @@
             "deckLimit": 3,
             "illustrator": "Andreia Ugrai"
         },
+        {
+            "code": "14020",
+            "type": "event",
+            "name": "Azor Ahai Reborn",
+            "faction": "baratheon",
+            "loyal": true,
+            "cost": 3,
+            "traits": [
+                "Legacy",
+                "R'hllor"
+            ],
+            "text": "Unique character you control only. Terminal.\nAttached character is considered to be participating in each challenge in which you control an attacking <i>R'hllor</i> character.",
+            "deckLimit": 3,
+            "illustrator": "Tim Durning"
```

Comment:
should be unique.

Reply:
Thanks for the catch! It should also be an attachment, not an event.